nearpy/tests/engine_tests.py

```diff
@@ -73,5 +73,69 @@ def test_retrieval_sparse(self):
             self.assertEqual(y_data, x_data)
             self.assertAlmostEqual(y_distance, 0.0, delta=delta)
 
+    def test_delete_vector(self):
+        # test case for 1 hash
+        dim = 5
+        unibucket = UniBucket('testHash')
+        engine = Engine(dim, lshashes=[unibucket])
+        # Index 20 random vectors (set their data to a unique string)
+        for index in range(20):
+            v = numpy.random.randn(dim)
+            engine.store_vector(v, index)
+        # Create random query vector
+        keys = set()
+        for lshash in engine.lshashes:
+            for value in engine.storage.buckets[lshash.hash_name].items():
+                [keys.add(id) for v, id in value[1]]
+
+        self.assertEqual(len(keys), 20)
+
+        engine.delete_vector(15) #delete the vector with key = 15
+
+        new_keys = set()
+        for lshash in engine.lshashes:
+            for value in engine.storage.buckets[lshash.hash_name].items():
+                [new_keys.add(id) for v, id in value[1]]
+
+        self.assertTrue(len(keys) > len(new_keys))
+        self.assertEqual(len(new_keys), 19) #new keys has 19 elements instead of 20
+
+        self.assertTrue(15 in keys)
+        self.assertFalse(15 in new_keys) # the key 15 is the one missing
+
+    def test_delete_vector2(self):
```

Comment:
This name is not very descriptive. Move your comment from the next line into the test name.